streaming-binlog/src/test/java/org/openmrs/analytics/UuidUtilTest.java

```diff
@@ -0,0 +1,103 @@
+
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.when;
+
+import java.beans.PropertyVetoException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+
+import junit.framework.TestCase;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.openmrs.analytics.model.EventConfiguration;
+
+@RunWith(MockitoJUnitRunner.class)
+public class UuidUtilTest extends TestCase {
+	
+	@Mock
+	private JdbcConnectionUtil jdbcConnectionUtil;
+	
+	@Mock
+	private Statement statement;
+	
+	@Mock
+	private ResultSet resultset;
+	
+	private EventConfiguration config;
+	
+	private Map<String, String> payLoad;
+	
+	private String uuid;
+	
+	private String table;
+	
+	private String keyColumn;
+	
+	private String keyValue;
+	
+	@Before
+	public void beforeTestCase() throws Exception {
+		payLoad = new HashMap<>();
+		config = new EventConfiguration();
+		
+		payLoad.put("uuid", "1");
+		payLoad.put("sex", "male");
+		payLoad.put("patient_id", "5");
```

Comment:
What is the point of `payload` and the values you `put` in it? Can't you directly set `keyValue` to `5` on line 76 and get rid of `payload`?